src/screens/AboutApp/AboutApp.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { ScrollView, View, Text, Image } from "react-native";
+import { translate } from "../../i18n";
+import { Button } from "../../components";
+
+import styles from "./styles";
+
+const logo = require("../../assets/logotype/Logo.png");
+
+const AboutApp = ({ version, phone, email }) => (
+  <ScrollView showsVerticalScrollIndicator={false} style={styles.container}>
+    <View style={styles.logo}>
+      <Image resizeMode="contain" style={styles.image} source={logo} />
+      <Text style={styles.version}>
+        {`${translate("aboutApp.version")} ${version}`}
+      </Text>
+    </View>
+    <Text style={styles.text}>{translate("aboutApp.description")}</Text>
+    <Button
+      caption={translate("aboutApp.readMore")}
+      appearance="Light"
+      wrapperStyles={styles.buttonWrapper}
+      borderColorStyle={styles.signUpBorder}
+      onPress={() => {}}
```

Comment:
I think some link should be provided to be opened

Reply:
its will be fixed when we will know about  that